tests/queries/0_stateless/00396_uuid.sql

```diff
@@ -11,3 +11,13 @@ with generateUUIDv4() as uuid,
     identity(lower(hex(reverse(reinterpretAsString(uuid))))) as str,
     reinterpretAsUUID(reverse(unhex(str))) as uuid2
 select uuid = uuid2;
+
+with generateUUIDv4() as uuid,
```

Comment:
@FrankChen021 better to add new test, without modifying old one and fix it. Please check CI report.

Reply:
Let me check why it fails. The test passed before I opened the PR.

Reply:
I found the answer. Original hex/bin functions don't output the leading zero for integers, so if each 64bit of UUID has leading zeros, they're also skipped. This might lead to test failure because UUID is generated randomly, and in that case, I think the generated UUID contains leading zeros.

For UUID's hex output, the leading zero should also be output.

Reply:
@FrankChen021 can't we use string implementation instead of trying to reuse integer implementations ?
Also new changes in tests should be moved to new tests, that way it is easier to track where tests start to fail

Reply:
We can't use the string implementation. Because we want to get the result in big-endian order, while the string implementation still generates a string in little-endian order.